sdcm/scylla_bench_thread.py

```diff
@@ -175,7 +175,7 @@ def _run_stress_bench(self, node, loader_idx, stress_cmd, node_list):
 
         log_file_name = os.path.join(node.logdir, f'scylla-bench-l{loader_idx}-{uuid.uuid4()}.log')
         # Select first seed node to send the scylla-bench cmds
-        ips = node_list[0].cql_ip_address
+        ips = ",".join([n.cql_ip_address for n in node_list])
 
         with ScyllaBenchStressExporter(instance_name=node.cql_ip_address,
```

Comment:
Shouldn't lines 180, 185 and 186 be changed too? These are node-specific.

Reply:
in those lines, the node is the loader.  (using cql_ip_address is not correct here, but not breaking anything)